aws-ssm-params/README.md

```diff
@@ -0,0 +1,42 @@
+# AWS SSM Params Reader
+
+This module is made to work together with [Chamber](https://github.com/segmentio/chamber) to manage secrets in AWS. Typically a user would use chamber to put secrets into the ParamStore and then use this module to read them out in Terraform code.
+
+You can use [our secrets setup module](../aws-param-secrets-setup/README.md) to prepare an AWS account/region to work with these tools.
+
+## Example
+
+```hcl
+module "secret" {
+  source = "github.com/chanzuckerberg/cztack/aws-ssm-params-secret?ref=v0.14.0"
+
+  project = "acme"
+  env     = "staging"
+  service = "website"
+
+  name = "password"
```

Comment:
```suggestion
  parameters = "password"
```
and `module.secret.values` in the output?

Reply:
Fixed.